pyproject.toml

```diff
@@ -33,3 +33,43 @@ asyncio_mode = "auto"
 
 [project.optional-dependencies]
 dev = ["documenteer[pipelines]"]
+
+[tool.towncrier]
```

Comment:
There is a config in ts_pre_commit_conf as well. Please use that instead. If you disagree, please explain why.